src-electron/db/query-config.js

```diff
@@ -25,7 +25,7 @@ const dbMapping = require('./db-mapping.js')
 const queryPackage = require('./query-package.js')
 const dbEnum = require('../../src-shared/db-enum.js')
 const queryZcl = require('./query-zcl.js')
-const queryUpgrade = require('../upgrade/upgrade.js')
+const queryUpgrade = require('../matter/matter.js')
```

Comment:
As a matter of general rules: we should NOT use the `.js` suffix in require calls.
That allows you to migrate the specific file to a different language (like TS) without having to modify these requires.